packages/react-examples/src/react/__snapshots__/TagPicker.CustomIcon.Example.tsx.shot

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@behowell I am getting `api-extractor` not updated issues currently. As the file has been renamed and a new shot has been generated, should I remove this? Would this fix the build?